fhir-parser-resources/mappings.py

```diff
@@ -80,17 +82,30 @@
 # For enum codes where a computer just cannot generate reasonable names
 enum_map = {
     '=': 'eq',
+	'!=': 'neq',
```

Comment:
I'd be more in favor of 'ne' rather than 'neq', but I suppose that's a matter of taste ;)

Reply:
Good catch, I'll make this change to 'ne'.  This is also consistent with FHIR search terms, http://hl7.org/fhir/search.html#prefix